support-frontend/assets/components/thankYou/thankyouModules.tsx

```diff
@@ -25,16 +25,20 @@ const firstColumnContainer = css`
 
 export interface ThankYouModulesProps {
 	isSignedIn?: boolean;
+	showNewspaperArchiveBenefit?: boolean;
```

Comment:
Super minor but would it simplify things to have this be required but with a default (of `false`)? That way within the component you know it's only ever going to be true or false and not undefined.

Reply:
Hmmm, your right, this module was called only so it did not need to be optional, have changed to `boolean` so cannot get `undefined`. I think as soon as I declare it as boolean it will default to false (is that ok?) but have made it explicit anyway here. Thanks for tips, perhaps should reduce the use of optional parameters in general?